tests/fields/test_identifier_field.py

```diff
@@ -45,37 +45,28 @@ def test_valid_identifiers():
 
 
 def test_invalid_duplicate_identifiers():
-    duplicate_identifiers = {
-        "identifiers": [{
-            "identifier": "10.5281/zenodo.9999999",
-            "scheme": "doi"
-            }, {
-            "identifier": "10.5281/zenodo.9999999",
-            "scheme": "doi"
-        }]
-    }
-
-    with pytest.raises(ValidationError):
-        TestSchema().load(duplicate_identifiers)
-
-
-def test_invalid_duplicate_identifiers_no_scheme_provided():
-    duplicate_identifiers = {
-        "identifiers": [{
-            "identifier": "10.5281/zenodo.9999999",
-            }, {
-            "identifier": "10.5281/zenodo.9999999",
-        }]
-    }
-
-    with pytest.raises(ValidationError):
-        TestSchema().load(duplicate_identifiers)
-
-
-def test_invalid_empty_identifiers():
```

Comment:
This test was actually raising a different error:
```
'identifier': ['Missing required identifier.']
```

Which is tested at `IdentifierField` level, not schema.